opa/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'Open Academy',
+    'summary': 'Manage your academy.',
+    'author': 'Vauxoo',
+    'website': 'https://vauxoo.com',
+    'category': 'Services',
+    'version': '15.0.0.1.0',
+    'depends': ['base'],
+    'data': [
+        'views/course_views.xml',
+        'views/session_views.xml',
+        'views/res_partner_views.xml',
+        'data/ir_ui_menu.xml',
+        'data/res_partner_category.xml',
+    ],
+    'demo': ['data/opa_demo.xml'],
```

Comment:
Demo data should be placed under the demo directory

Reply:
I left it there because according to [the guidelines](https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#module-structure) demo files should be on the data directory.
Should I still it move it?

Reply:
@antonag32 We are more aligned with OCA guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#14directories

Please attend the comment

Reply:
I see, thanks, I'll change it.